src/sentry/utils/contexts_normalization.py

```diff
@@ -24,10 +27,16 @@ def normalize_os(data):
             data['name'] = 'Windows'
             data['version'] = r.group('version')
         else:
-            r = _uname_re.search(raw_description)
+            r = _unreal_macos_re.search(raw_description)
```

Comment:
This code is already phasing out and has moved to [libsemaphore](https://github.com/getsentry/semaphore/blob/master/general/src/store/normalize/contexts.rs). Could you create a PR that adds the same logic there?